cmake/PythonCopyStandardLib.py

```diff
@@ -90,35 +89,33 @@
     # Let's check the library files to see if the ABI matches
     # Otherwise if you build with say python 3.8 initially, and then switch to
     # python 3.9, your lib-dynload will still have the 38 .so files
-    std_lib_ctypes_sos = gb.glob(os.path.join(standard_lib_dir, "**/_ctypes.*"), recursive=True)
-    this_lib_ctypes_sos = gb.glob(os.path.join(target_dir, "**/_ctypes.*"), recursive=True)
 
-    def find_libs(dirPath):
+    def find_libs(dir_path):
         sos = []
         for ext in ['a', 'so', 'lib']:
-            sos += gb.glob(os.path.join(dirPath, "**/*.{}*".format(ext)),
+            sos += gb.glob(os.path.join(dir_path, "**/*.{}*".format(ext)),
                            recursive=True)
         return [os.path.basename(f) for f in sos]
 
+
     std_lib_ctypes_sos = find_libs(standard_lib_dir)
     this_lib_ctypes_sos = find_libs(target_dir)
 
     if ((set(std_lib_ctypes_sos) - set(this_lib_ctypes_sos)) or
-        (set(this_lib_ctypes_sos) - set(std_lib_ctypes_sos))):
+            (set(this_lib_ctypes_sos) - set(std_lib_ctypes_sos))):
         print("Detected changes in the python libs, wiping and recopying")
         shutil.rmtree(target_dir)
     else:
         # File names match
         sys.exit(0)
 
-
 shutil.copytree(standard_lib_dir, target_dir)
 
 # On Windows, we also need to grab the DLLs folder, which is one folder up
 if platform.system() == 'Windows':
     python_root_dir = os.path.dirname(standard_lib_dir)
     dll_dir = os.path.join(python_root_dir, 'DLLs')
-    copy_tree(dll_dir, target_dir)
+    shutil.copytree(dll_dir, target_dir, dirs_exist_ok=True)
```

Comment:
Use `shutil` version of copytree instead of the `distutils` imported version of it.